llama/llama-2-7b-trt-llm/model/model.py

```diff
@@ -61,7 +68,11 @@ def predict(self, model_input):
         bad_words_list = model_input.get("bad_words_list", [""])
         stop_words_list = model_input.get("stop_words_list", [""])
         repetition_penalty = model_input.get("repetition_penalty", 1.0)
-        end_id = model_input.get("end_id", 2)  # EOS token for Llama2
+        if not model_input.get("ignore_eos", False):
+            end_id = model_input.get("end_id", self.eos_token_id)
+        else:
+            # do nothing, trt-llm by default doesn't stop on `eos`
+            pass
```

Comment:
In this case isn't `end_id` undefined but we attempt to prepare and pass this tensor to the server?

Reply:
Yeah, it was something weird written before, thanks for the catch!